src/brain/projectsHandler/project.ts

```diff
@@ -60,7 +76,7 @@ export async function syncLabelsWithProjectField({
     owner,
     repo: issueInfo.repo,
     issue_number: issueInfo.number,
-    labels: [`${PRODUCT_AREA_LABEL_PREFIX}${productArea}`],
+    labels: [`${fieldName === "Product Area" ? PRODUCT_AREA_LABEL_PREFIX : ""}${fieldValue}`],
```

Comment:
Feels weird to have this special case. Don't "Waiting for: Foo" labels also have a prefix?

Reply:
I agree, feels weird to have. As of right now, single select options under the `Product Area` field in the project appear as `Foo, Bar, etc...`. To make it more consistent, we could add `Product Area: Foo` as options in the Product Area field, but that seems a bit redundant?